trade-finance-manager-api/api-tests/v1/users/users.api-test.js

```diff
@@ -11,6 +11,11 @@ describe('user controller', () => {
     tokenUser = await testUserCache.initialise(app);
   });
 
+  it('should not create a new TFM user with malformed payload', async () => {
+    const { body } = await as(tokenUser).post({}).to('/v1/users');
+    expect(body).toEqual({});
```

Comment:
It is because of the callback() which then throws an error on UI using `next()`.
This is a separate PR where all of these callback do throw back an error message.